### DISCUSSION TITLE
how to uninstall oh-my-posh in macOS?

### DISCUSSION THREAD

**Discussion by laptopmutia** [JanDeDobbeleer/oh-my-posh#6025] [Troubleshoot] [answered]
I really like this framework, but it takes 1 second to load the terminal, I find it not worth it at all for my use case,
therefore I want to uninstall, but there is no documentation about uninstalling, anyone can help me?

I only found how to uninstall on powershell module

**Comment by JanDeDobbeleer** [comment]
@laptopmutia if you're using PowerShell, that's PowerShell being slow, not Oh My Posh. I suggest migrating to a faster shell like `nushell` or `zsh`.

**Reply by laptopmutia** [reply]
sorry forgot to mention that I use macbook pro with m3 pro chipset
and macOS

I also use iterm2 and zsh shell,

can you help me with the unistall steps?

**Reply by JanDeDobbeleer** [reply]
how did you install? Because that's either `homebrew` or manual. The uninstallation steps differ.

**Reply by laptopmutia** [reply]
I use `homebrew`

**Reply by JanDeDobbeleer** [reply, marked as answer]
then just `brew uninstall` and remove the entry from your `.zshrc`

**Reply by laptopmutia** [reply]
thanks, really appreciate the help
